lib/runtime/wasmer/instance.go

```diff
@@ -187,6 +187,11 @@ func (in *Instance) GetCodeHash() common.Hash {
 	return in.codeHash
 }
 
+// GetContext returns the context of the instance
+func (in *Instance) GetContext() *runtime.Context {
```

Comment:
Seems odd that we need to modify the context only in tests

Reply:
Agreed, I needed it to create valid extristics based on the current/correct runtime context. I think its modified elsewhere in the package, it just is not exported